srcpkgs/perl-Math-Int128/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'perl-Math-Int128'
+pkgname=perl-Math-Int128
+version=0.22
+revision=1
+# not supported on 32-bit architectures
+archs="aarch64* ppc64* x86_64*"
```

Comment:
You can do a conditional on `$XBPS_TARGET_WORDSIZE = 64` and mark the others as broken, mkes it simpler.

Reply:
Yes, that’s much better, thank you. :)